checks/batch/__init__.py

```diff
@@ -313,7 +317,38 @@
                 ]
             }
         ]
-    }
+    },
+    {
+        'name': 'web_rpki',
+        'type': 'category',
+        'translation_key': 'siterpki',
+        'children': [
```

Comment:
You forgot the sections that you have on the result page. Then the tests here are children of the sections. This is mostly for API users that want to replicate the result page as it appears on internet.nl.